### ISSUE TITLE
Create and delete files inside folder, but not delete folder?

### ISSUE THREAD

**Issue by ext-caucesystem** [nextcloud/groupfolders#605]
There is any way of giving users the right to create and delete files inside a folder, but not to delete the folder itself?

**Comment by wiswedel** [comment]
Not that I'm aware of
CC @juliushaertl @icewind1991

**Comment by tobiasfuhlroth** [comment]
The parent folders permission should determine if a user is able to delete a folder or a file inside the parent folder. If the folder itself gets affected by the permission, there is no way to prevent it from being deleted by a user.

**Comment by juliushaertl** [comment]
Not possible, since the permissions always apply to the folder they are set on as well as to the children if not overwritten by child rules.

**Comment by ouvry-ems** [comment]
Hi,
Is there any news on this topic? 
It's indeed a big need to allow permissions on children only and not on the folder/parent.
Cheers!

**Comment by Jerome-Herbinet** [comment]
I agree with @ext-caucesystem and @ouvry-ems and I think this request is absolutely relevant !

Granting delete permissions within a folder (content) has a very different impact than applying them to the folder itself (container). 

In real life (versus theory), and quite legitimately, there are a whole host of reasons and scenarios in which a team folder administrator might want to allow people to delete files within a folder, but not the folder itself.

This additional granularity would provide a real improvement in terms of folder longevity for team folder managers.
